fastlane/Fastfile

```diff
@@ -1476,15 +1472,13 @@ TEST_ANALYTICS_ENVIRONMENT = %w[
 # -----------------------------------------------------------------------------------
 # Release Management Utils
 # -----------------------------------------------------------------------------------
-def create_release_management_pull_request(base_branch:, title:)
-  create_pull_request(
-    api_token: get_required_env('GITHUB_TOKEN'),
-    repo: GITHUB_REPO,
-    title: title,
-    head: Fastlane::Helper::GitHelper.current_git_branch,
-    base: base_branch,
-    labels: 'Releases'
-  )
+def create_backmerge_pr(version:)
+    create_release_backmerge_pull_request(
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/woocommerce/woocommerce-ios/blob/eac4c120e43662f6facce5d4552f998b782be980/fastlane/Fastfile#L1476"></span>Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.</td>
    </tr>
  </tbody>
</table>